conf/application.conf

```diff
@@ -195,6 +195,11 @@ gform-frontend-base-url = "http://localhost" # local nginx
 agent-subscription-frontend-base-url = "http://localhost:9437"
 signout-url="https://www.tax.service.gov.uk/gg/sign-out"
 footer-accessibility-statement-url="/accessibility-statement"
+pta-home-url="/personal-account"
+pta-messages-url="/personal-account/messages"
+pta-track-url="/track"
+pta-profile-url="/personal-account/your-profile"
```

Comment:
Are these going to change per environment? If not then what is the point, why not just hardcoded these to make code easier to read.

Reply:
Updated to hardcoded values